pkg/ginkgo/util.go

```diff
@@ -81,16 +84,36 @@ func BuildExtensionTestSpecsFromOpenShiftGinkgoSuite(selectFns ...ext.SelectFunc
 					Name: spec.Text(),
 				}
 
+				// Keep Verbose = true to capture g.By() output
+				// We'll filter out the reporter summary from the captured output later
+				reporterConfigCopy := *reporterConfig
+				reporterConfigCopy.Verbose = true
+
+				// Create a buffer to capture g.By() output AND write to stderr for real-time visibility
+				var outputBuffer bytes.Buffer
+				multiWriter := io.MultiWriter(&outputBuffer, os.Stderr)
```

Comment:
Here simplify by Using Ginkgo's Built-in Separation, the current approach creates a 40-line filter function, but Ginkgo already separates test output from reporter output! 
```
// Just capture to buffer for stderr visibility
  var outputBuffer bytes.Buffer
  multiWriter := io.MultiWriter(&outputBuffer, os.Stderr)
  captureWriter := ginkgo.NewWriter(multiWriter)

  ginkgo.GetSuite().RunSpec(spec, ..., captureWriter, ...)

  // Use Ginkgo's pre-separated output - NO FILTERING NEEDED!
  result.Output = summary.CapturedGinkgoWriterOutput  // Already clean!
  result.Error = summary.CapturedStdOutErr

```